experimental/packages/exporter-trace-otlp-http/src/platform/node/util.ts

```diff
@@ -88,6 +88,7 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
         gzip = zlib.createGzip();
       }
       req.setHeader('Content-Encoding', 'gzip');
+      req.removeHeader('Content-Length');
```

Comment:
I think it would be more intuitive to add `Content-Length` only in the non compressed case instead of removing it in all compressed variants.
As of now it's a 1:1 but once more compression algorithms are added we end up in a lot remove calls.

Reply:
Done.